src/tribler-gui/tribler_gui/__init__.py

```diff
@@ -9,6 +9,11 @@
 
 logger = logging.getLogger(__name__)
 
+
+class CoreError(Exception):
+    """This is the base class for exceptions that causes GUI shutdown"""
```

Comment:
@kozlovsky I'm not sure that `__init__.py` is the best place for the `CoreError` class. If you have any suggestions feel free to give them to me.

Reply:
I think we should have `exceptions.py` file in `tribler_gui` similar to `exceptions.py` file in `tribler_core`

Reply:
exceptions.py has been added.